drivers/lpsxxx/Kconfig

```
@@ -6,38 +6,42 @@
 # directory for more details.
 #
 
-choice
-    bool "LPSXXX Pressure Sensors"
-    optional
+menuconfig MODULE_LPSXXX
+    bool
+    prompt "LPSXXX Pressure Sensors" if !(MODULE_SAUL_DEFAULT && HAVE_LPSXXX)
     depends on HAS_PERIPH_I2C
     depends on TEST_KCONFIG
+    select MODULE_PERIPH_I2C
+    help
+        Device driver for the LPSXXX pressure sensor family
+        (LPS331AP/LPS25HB/LPS22HB/LPS22HH). Select a model.
+
+if MODULE_LPSXXX
+
+choice LPSXXX_VARIANT
+    bool "Model"
+    default MODULE_LPS331AP if HAVE_LPS331AP
+    default MODULE_LPS22HB if HAVE_LPS22HB
+    default MODULE_LPS22HH if HAVE_LPS22HH
+    default MODULE_LPS25HB if HAVE_LPS25Hb
```

Sorry, just spotted this one
```suggestion
    default MODULE_LPS25HB if HAVE_LPS25HB
```